test/modules/core-layers/core-layers.spec.js

```diff
@@ -143,15 +143,6 @@ test('ArcLayer#constructor', t => {
           getSourcePosition: d => d.START,
           getTargetPosition: d => d.END
         }
-      },
-      {
-        updateProps: {
```

Comment:
Hard to port these test cases to the new world?

Reply:
We are no longer setting these props as uniforms.